app/workers/rapid_ready_for_decision/disability_compensation_job.rb

```diff
@@ -31,7 +31,12 @@ def perform(form526_submission_id)
 
           return if bp_readings(client).blank?
 
-          pdf = pdf(form526_submission.full_name, bp_readings(client), medications(client))
+          patient_info = {
+            **form526_submission.full_name,
+            va_eauth_birthdate: form526_submission.auth_headers['va_eauth_birthdate']
+          }
```

Comment:
@yoomlam @nanotone so, this could happen in form526_submission as well, but I'm trying to avoid making RRD changes there. This should probably be a new method.